client/src/components/publications/form/PageOne.js

```diff
@@ -1,47 +1,42 @@
+/**
+ * The PageOne component displays the first page of the publication form
```

Comment:
`PageOne` can be renamed to something more descriptive? If the order of pages change, the naming will no longer reflect what the component displays.

Reply:
Agree. Will change it.